Visual mode cannot be navigated with up/down arrow keys

**Describe the bug**
Visual mode cannot be navigated with up/down arrow keys. I guess this is a good time to get used to h/j/k/l :sweat_smile: 

**To Reproduce**
Steps to reproduce the behavior:

1.  Press v to enter visual mode
2.  Move left/right with arrow keys, works. 
3.  Move up/down with arrow keys, exits visual mode into normal mode

- Extension (VsCodeVim) version: 1.15.0
- VSCode version: 1.47.1
- OS: Ubuntu

@berknam It occurred randomly once again. Following are the outputs for the inspect context keys object:
```json
editorHasSelection: false
editorTextFocus: true
parameterHintsMultipleSignatures: true
suggestWidgetVisible: false
vim.active: true
vim.mode: "Normal"
vim.use<down>: true
vim.use<up>: true
```

Also got thee following error when trying Inspect Context Keys. Not sure if helpful.
```bash
Activating extension 'vscode.extension-editing' failed: Cannot find module '/snap/code-insiders/482/usr/share/code-insiders/resources/app/extensions/extension-editing/dist/extensionEditingMain'
Require stack:
- /snap/code-insiders/482/usr/share/code-insiders/resources/app/out/vs/loader.js
- /snap/code-insiders/482/usr/share/code-insiders/resources/app/out/bootstrap-amd.js
- /snap/code-insiders/482/usr/share/code-insiders/resources/app/out/bootstrap-fork.js.
```

--- a/package.json
+++ b/package.json
@@ -113,12 +113,12 @@
       {
         "key": "up",
         "command": "extension.vim_up",
-        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !suggestWidgetMultipleSuggestions && !parameterHintsMultipleSignatures"
+        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !parameterHintsVisible"
       },
       {
         "key": "down",
         "command": "extension.vim_down",
-        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !suggestWidgetMultipleSuggestions && !parameterHintsMultipleSignatures"
+        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !parameterHintsVisible"
       },
       {
         "key": "g g",

